test/definitions/super-agent/suse/suse155-super-agent.json

```diff
@@ -0,0 +1,35 @@
+{
```

Comment:
Recipes changes are done in the super-agent/logs, hence the test definition configs needs to be created for super-agent logs and not for super-agent.

At the moment the changes are done here `test/definitions/super-agent/` instead these should be done at `test/definitions/super-agent/logs`

Reply:
updated PR